server/index.js

```diff
@@ -1,53 +1,19 @@
-const express = require('express');
-const db = require('../db/index.js');
-const Bundles = require('../db/models/bundles.model.js');
+const db = require('../db/index');
+const Bundles = require('../db/models/bundles.model');
+const app = require('./app');
 
-const app = express();
 const port = 3663;
 
-app.use(express.urlencoded({
-  extended: true,
-}));
-
-db.sync({ force: false })
+db.sync({
+  force: false,
+  logging: false
+})
   .then(() => {
     console.log('db synced: ', db);
     app.listen(port, () => {
       console.log(`dockside at port ${port}`);
     })
   })
-  .then(() => {
-    Bundles.findOne({
-      where: {
-        id: 100,
-      },
-    })
-    .then((results) => {
-      console.log('results: ', results.dataValues);
-    });
-  })
   .catch((err) => {
     console.log('your error: ', err);
   });
```

Comment:
Great separation with your db/index file! I could be blind but I'm not seeing the use of the Bundles const on line 2. If it is ignore this comment lol.

Reply:
Deleted.